docs/ReferenceChanges.md

```diff
@@ -0,0 +1,50 @@
+Package: Microsoft.Extensions.Configuration.Binder
+  TFM: .NETStandard2.0
+    Removed:
+      Microsoft.Extensions.Configuration
+Package: Microsoft.Extensions.Configuration.Json
+  TFM: .NETStandard2.0
+    Removed:
+      System.Threading.Tasks.Extensions
+Package: Microsoft.Extensions.DependencyInjection
+  TFM: .NETFramework4.6.1
+    Added:
+      System.Threading.Tasks.Extensions
```

Comment:
I just checked, these were in the previous closure as well since Microsoft.Bcl.AsyncInterfaces included a reference to it so this is a false positive. I didn't include that package in the set of packages I was analyzing. The same goes for M.E.Hosting.Abstraction.